runtime/vscode-neovim.vim

```diff
@@ -2,12 +2,41 @@
 let g:vscode = 1
 
 let s:currDir = fnamemodify(resolve(expand('<sfile>:p')), ':h')
-" Adjust rtp path
-let &runtimepath = &runtimepath . ',' . s:currDir . '/vim-altercmd'
+let &runtimepath = &runtimepath . ',' . s:currDir
+
+" Load atltercmd
+" {{{
+" altercmd - Alter built-in Ex commands by your own ones
+" Version: 0.0.1
+" Copyright (C) 2009-2015 Kana Natsuno <http://whileimautomaton.net/>
+" License: MIT license
+function! s:altercmd_define(...)
```

Comment:
interesting, did you inline the only part of AlterCmd that we need? so that allowed us to delete the rest of it? that is something worth mentioning in the PR description & commit message.

Reply:
altercmd has only this piece of code 😂